Лабораторная работа №1/Замена пропущенного элемента средним арифметическим.py

```diff
@@ -0,0 +1,7 @@
+numbers = [2, -93, -2, 8, None, -44, -1, -85, -14, 90, -22, -90, -100, -8, 38, -92, -45, 67, 53, 25]
+index_of_None = 4
```

Comment:
Далее, когда вы разберете методы, искать индекс можно будет не самостоятельно, а используя метод для поиска элемента в списке. Тогда эта строка будет выглядеть следующим образом: 
```suggestion
index_of_None = numbers.index(None)
```